Few-iOS/TableView.swift

```diff
@@ -78,17 +120,31 @@ private class TableViewHandler: NSObject, UITableViewDelegate, UITableViewDataSo
 		self.headers = headers
 		self.footers = footers
 		super.init()
-		tableView.registerClass(FewListHeaderFooter.self, forHeaderFooterViewReuseIdentifier: headerKey)
-		tableView.registerClass(FewListHeaderFooter.self, forHeaderFooterViewReuseIdentifier: footerKey)
+		headerView.didChangeHeight = {[weak tableView] view in
+			if let tableView = tableView where tableView.tableHeaderView == view {
+				tableView.tableHeaderView = view
```

Comment:
We need `UITableView` to update its row positioning for the new header height but it doesn't observe the header frame change, so we nudge it.